homebrew_challenge/src/components/GeoHeatMap/RegionTypeSelector.jsx

```diff
@@ -4,7 +4,7 @@ import ToggleButtonGroup from "react-bootstrap/ToggleButtonGroup";
 import "../../common.css";
 import {
   AREATYPE_COUNCIL_AREAS,
-  AREATYPE_HEALTH_BOARDS,
+  AREATYPE_HEALTH_BOARDS
```

Comment:
It looks like your autoformat settings are different from mine. Mine always leaves a trailing comma

Reply:
added